apps/health_campaign_field_worker_app/lib/pages/beneficiary_registration/custom_individual_details.dart

```diff
@@ -487,8 +501,8 @@ class CustomIndividualDetailsPageState
                                       Flexible(
                                         child: Text(
                                           overflow: TextOverflow.ellipsis,
-                                          localizations
-                                              .translate(state.qrCodes.first),
+                                          localizations.translate(
+                                              state.qrCodes.toString()),
```

Comment:
qrCodes is an array right? is this working fine?

Reply:
this is fixed in 2nd pr, you can merge this